documentation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Get Started | gotham UI</title>
```

Comment:
Better to add meta tag with your name and also for description.

Reply:
yup, will do that for all components, I will do it by creating a branch